models/brac/useviews/chwview_assessment.sql

```diff
@@ -82,13 +78,12 @@ FROM(
            WHEN ((form.doc #>> '{fields,patient_age_in_years}'::text[])::integer) < 5 AND (form.doc #>> '{fields,referral_follow_up}'::text[]) = 'true'::text THEN 1
            ELSE 0
        END) AS required_follow_ups
-  FROM {{ ref("couchdb") } form
- WHERE (form.doc ->> 'type'::text) = 'data_record'::text AND form.doc ? 'form'::text AND (form.doc ->> 'form'::text) = 'assessment'::text
+  FROM {{ ref("couchdb") }} form
+ WHERE (form.doc ->> 'type'::text) = 'data_record'::text AND (form.doc ->> 'form'::text) = 'assessment'::text
```

Comment:
I'm unsure of how to deal with this view since it's an aggregation view rather than containing raw data